js/menu.js

```diff
@@ -873,7 +873,14 @@ var referencePane = {
 
   showSDOs: function(sdos, alternativeId) {
     this.activate();
-    this.$headerText.textContent = 'Syntax-Directed Operations';
+    var rhs = document.getElementById(alternativeId);
+    var parentName = rhs.parentNode.getAttribute('name')
+    rhs = rhs.cloneNode(true);
+    rhs.querySelectorAll('emu-params,emu-constraints').forEach(function (e) { e.remove(); });
+    rhs.querySelectorAll('emu-opt').forEach(function (e) { e.innerHTML = '_opt' });
```

Comment:
I don't like this rendering. I would much prefer these still be rendered as subscripts.

<img width="1165" alt="Screenshot 2021-01-23 at 22 52 13" src="https://user-images.githubusercontent.com/218840/105623322-ab62f480-5dcd-11eb-9c60-64a1c5f49de6.png">